### PR TITLE
Update SerialWombat_mp_i2c.py to use super().__init__()

### PR DESCRIPTION
**Pull request by crecine**
If I'm understanding the intent of this line, I think this is the preferred method to initialize.
using super() to init the micropython I2C chip

### CODE REVIEW

**Comment by BroadwellConsultingInc** [comment]
That is correct.  I wish to call the parent's __init__ call .   There are many places where I think this change probably needs to be made.

File: interfaces/micropython/SerialWombat_mp_i2c.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ class SerialWombatChip_mp_i2c(SerialWombat.SerialWombatChip):
     def __init__(self,i2c_port,address):
         self.i2c = i2c_port
         self.address = address
-        SerialWombat.SerialWombatChip.__init__(self)
+        super().__init__()
```
Comment (crecine) [inline interfaces/micropython/SerialWombat_mp_i2c.py:12]: In that case, the SerialWombatChip's init is setting self.address to 0, which overwrites the value set here on line 11.
If you change this to:
```
    def __init__(self,i2c_port,address):
        super().__init__()
        self.i2c = i2c_port
        self.address = address
```
you won't have that problem

Comment (BroadwellConsultingInc) [reply interfaces/micropython/SerialWombat_mp_i2c.py:12]: I noticed that as well.  I fixed it after I merged your pr.   Thanks again!

Comment (crecine) [reply interfaces/micropython/SerialWombat_mp_i2c.py:12]: Great! I'm glad I could help, I appreciate all the work you've put into this so far.

**Pull request merged**